microsetta_interface/templates/new_participant.jinja2

```diff
@@ -108,6 +103,57 @@
         disableInputOnSubmit("#7-12-form", "#7-12-submit_button", "{{ _('Saving...') }}");
         disableInputOnSubmit("#13-17-form", "#13-17-submit_button", "{{ _('Saving...') }}");
         disableInputOnSubmit("#18-form", "#18-submit_button", "{{ _('Saving...') }}");
+        $("#home_page").click(function(){
+            window.open('{{home_url}}', '_self')
+        })
+        $("#create_source").click(function(){
+            //window.open('{{post_url}}', '_self')
+            //$("#source_modal").modal('hide');
+            $("#18-form")[0].submit();
+        })
+        $("#18-form").submit(function(e){
```

Comment:
This needs to work for all four types of consent: 0-6, 7-12, 13-17, and 18+. Currently it only works for 18+.